src/ethereum-providers/icons/walletConnect.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's optimize this file, not sure if you know how to do it but in. case you don't you need to get the library svgo and run `svgo .... file path`